doc/release-notes/release-notes-bucash1.2.0.0.md

```diff
@@ -0,0 +1,139 @@
+Release Notes for Bitcoin Unlimited Cash Edition 1.2.0.0
+=========================================================
+
+Bitcoin Unlimited Cash Edition version 1.2.0.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a minor release version based of Bitcoin Unlimited compatible
+with the Bitcoin Cash specification you could find here:
+
+https://github.com/Bitcoin-UAHF/spec/blob/master/uahf-technical-spec.md
+
+
+Upgrading
+---------
+
+If you are running an older version, shut it down. Wait until it has completely
+shut down (which might take a few minutes for older versions), then run the
+installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
+bitcoind/bitcoin-qt (on Linux).
+
+The first time you run this new version, your UTXO database will be converted
+to a new format. This step could take a variable amount of time that will depend
+on the performance of the hardware you are using.
+
+Downgrade
+---------
+
+In case you decide to downgrade from BUcash 1.2.0.0 to a previous version you
+will need to run the old release using `-reindex-chainstate` option so that the
+UTXO will be rebuild using the previous format.
+
+Main Changes
+------------
+
+The main changes of this release is the introduction of the cashaddr new address format.
+The specification for this new address encoding could be find [here](https://github.com/Bitcoin-UAHF/spec/blob/master/cashaddr.md).
+The old format continue to be supported and won't be deprecated, The old format will remain the default till January 14th, 2017, on
+that day the new format will be used as default new format automatically.
+A command line flag `-usecashaddr` and configuration parameter could be used to select which format to by default.
+
+Other notable changes:
+
+- Activate the new DAA by height instead of MTP (removal of `newdaaactivationtime` parameter)
+- Cash net magic became mandatory. Peers incoming connections using the old magic would be rejected.
+- Add limited support to BitPay Bitcoin Cash address [format](https://support.bitpay.com/hc/en-us/articles/115004671663-BitPay-s-Adopted-Conventions-for-Bitcoin-Cash-Addresses-URIs-and-Payment-Requests)
+- Coin cache improvements
```

Comment:
I would say what the rpc command is, "getaddressforms <address>" which returns all formats for displaying this address, so that it's clear what the support is...

Reply:
agreed

Reply:
https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/894.patch

Reply:
fixed while mving BitPay reference in the `Major changes` section